myst_nbs/howto/howto_debugging.myst.md

```diff
@@ -6,108 +6,184 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3
+  display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
+(howto_debugging)=
 # How to debug a model
 
-There are various levels on which to debug a model. One of the simplest is to just print out the values that different variables are taking on.
+:::{post} July 11, 2016
```

Comment:
The date here should be the last execution date (roughly). From https://docs.pymc.io/en/latest/contributing/jupyter_style.html:

> The date should correspond to the latest update/execution date, at least roughly (it’s not a problem if the date is a few days off due to the review process before merging the PR). This will allow users to see which notebooks have been updated lately and will help the PyMC team make sure no notebook is left outdated for too long.